tests/agents/test_max_height.py

```diff
@@ -0,0 +1,50 @@
+import holodeck
+import uuid
+from copy import deepcopy
+
+from tests.utils.equality import almost_equal
+
+uav_config = {
+    "name": "test_max_height",
+    "world": "TestWorld",
+    "main_agent": "uav0",
+    "agents": [
+        {
+            "agent_name": "uav0",
+            "agent_type": "UavAgent",
+            "sensors": [
+                {
+                    "sensor_type": "LocationSensor",
+                }
+            ],
+            "control_scheme": 0,
+            "location": [0, 0, 2],
+            "max_height": 500 - 11.8283691 # in centimeters
+        }
+    ]
+}
+
+
+def test_higher_than_mh():
+    """Make sure the location sensor updates after a teleport. Also verifies that the coordinates for the teleport
+    command match the coordinates used by the location sensor
+    """
+    # binary_path = holodeck.packagemanager.get_binary_path_for_package("DefaultWorlds")
```

Comment:
Same thing as the default height test, clean this up so it works on installed worlds.